modules/mockk/src/commonMain/kotlin/io/mockk/impl/verify/UnorderedCallVerifier.kt

```diff
@@ -55,15 +55,17 @@ open class UnorderedCallVerifier(
         val allCallsForMock = stub.allRecordedCalls()
         val allCallsForMockMethod = stub.allRecordedCalls(matcher.method)
 
-        if(allCallsForMockMethod.size > 1 && matcher.args.any { it is CapturingSlotMatcher<*> }) {
+        val matchedCalls = allCallsForMockMethod.filter(matcher::match)
+
+        if(matchedCalls.size > 1 && matcher.args.any { it is CapturingSlotMatcher<*> }) {
```

Comment:
It's OK if there are multiple calls overall but only one of them matches and will be captured